sql/expression/filter-range.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020-2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expression
+
+import (
+	"fmt"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/types"
+)
+
+// NewRangeFilterExpr builds an expression that filters with the list of sql ranges and exprs.
+//
+// The length of each range must match the length of the expressions slice.
```

Comment:
(minor) It would be nice to give a little detail on the `exprs` parameter and what should be passed in, especially since this is an exported function.

Reply:
Added. Thanks